app/src/main/res/layout/fragment_example.xml

```diff
@@ -1,95 +1,100 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout
+<layout xmlns:tools="http://schemas.android.com/tools"
 	xmlns:android="http://schemas.android.com/apk/res/android"
 	xmlns:app="http://schemas.android.com/apk/res-auto">
 
 	<androidx.constraintlayout.widget.ConstraintLayout
 		android:layout_width="match_parent"
 		android:layout_height="match_parent"
+		android:background="#FFFFFF"
 		android:gravity="center"
-		android:orientation="vertical"
-		android:padding="@dimen/spacing_large">
+		android:orientation="vertical">
 
-		<TextView
-			android:id="@+id/title"
+		<ImageView
+			android:id="@+id/imageView7"
+			app:layout_constraintTop_toTopOf="parent"
+			android:layout_width="match_parent"
+			android:layout_height="200dp"
+			android:adjustViewBounds="true"
+			android:background="@drawable/login_cover"
+			android:scaleType="centerCrop"
+			android:src="@drawable/gradient" />
+
+		<ImageView
+			android:id="@+id/imageView3"
 			android:layout_width="wrap_content"
 			android:layout_height="wrap_content"
-			android:text="@string/example_title"
-			android:textSize="@dimen/font_medium_extra"
-			app:layout_constraintBottom_toTopOf="@id/usernameInput"
+			android:layout_marginTop="120dp"
 			app:layout_constraintEnd_toEndOf="parent"
 			app:layout_constraintStart_toStartOf="parent"
-			app:layout_constraintTop_toTopOf="parent"
-			app:layout_constraintVertical_bias="0" />
+			app:layout_constraintTop_toTopOf="@+id/imageView7"
+			app:srcCompat="@drawable/wolox_logo_large" />
 
 		<EditText
-			android:id="@+id/usernameInput"
-			android:layout_width="match_parent"
-			android:layout_height="wrap_content"
-			android:layout_marginLeft="@dimen/spacing_large_extra"
-			android:layout_marginRight="@dimen/spacing_large_extra"
+			android:id="@+id/email_login"
+			android:layout_width="320dp"
```

Comment:
Mmmm.... Edit text must match the width of the screen. Solve harcoded dimens below too.